app/src/main/java/org/mozilla/fenix/tabstray/FloatingActionButtonBinding.kt

```diff
@@ -29,7 +31,6 @@ class FloatingActionButtonBinding(
 
     @OptIn(ExperimentalCoroutinesApi::class)
     override fun start() {
-        setFab(store.state.selectedPage, store.state.syncing)
```

Comment:
Removed this since it's not needed if we set the action button visibility to `gone` by default.